src/Microsoft.PowerShell.Commands.Utility/commands/utility/Tee-Object.cs

```diff
@@ -110,6 +120,9 @@ protected override void BeginProcessing()
                 // Can't use set-var's passthru because it writes the var object to the pipeline, we want just
                 // the values to be written
             }
+            
+            // passthrough Encoding parameter
+            _commandWrapper.AddNamedParameter("Encoding", Encoding);
```

Comment:
This doesn't work for ` Tee-Object -Variable` because we assign Encoding parameter to Set-Variable cmdlet. Please see previous https://github.com/PowerShell/PowerShell/blob/b303c2f614b8e158179ef9e2c2fe73110dbfb654/src/Microsoft.PowerShell.Commands.Utility/commands/utility/Tee-Object.cs#L117-L121

Reply:
Didn't even notice this, thanks. I updated it so that the Encoding parameter is only used for the `File` and `LiteralFile` parameter sets.